# Question 3

## Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions.

### Data Description
- α == 0.05 (95% Confidence)
- Y == Discrete
- X == Discrete

### Since there are more than 2 variable we will perform Chi-Square test

In [1]:
import pandas as pd
import numpy as np
from scipy import stats

In [2]:
buy=pd.read_csv("BuyerRatio.csv")
buy

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

- H0 = Male,Female buyer rations are similar across all region
- H1 = Male,Female buer rations are not similar across all region

In [3]:
stats.chi2_contingency([buy["East"], buy["West"], buy["North"], buy["South"]])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

- pvalue of Chi-Square test is 0.66>α
## Hence H0 is accepted because all proportions are equal.